firebase-common/src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -428,11 +427,7 @@ protected FirebaseApp(Context applicationContext, String name, FirebaseOptions o
             Component.of(applicationContext, Context.class),
             Component.of(this, FirebaseApp.class),
             Component.of(options, FirebaseOptions.class),
-            LibraryVersionComponent.create(FIREBASE_ANDROID, ""),
-            LibraryVersionComponent.create(FIREBASE_COMMON, BuildConfig.VERSION_NAME),
-            kotlinVersion != null ? LibraryVersionComponent.create(KOTLIN, kotlinVersion) : null,
-            DefaultUserAgentPublisher.component(),
-            DefaultHeartBeatInfo.component());
+            kotlinVersion != null ? LibraryVersionComponent.create(KOTLIN, kotlinVersion) : null);
```

Comment:
This cannot be moved inside the new FirebaseCommonRegistrar? Since I am assuming the useragentstring will also get this value right?